front50-web/src/main/java/com/netflix/spinnaker/front50/controllers/PipelineController.java

```diff
@@ -267,7 +268,11 @@ private void validatePipeline(final Pipeline pipeline, Boolean staleCheck) {
         pipeline.getApplication(), pipeline.getName().trim(), pipeline.getId());
 
     final GenericValidationErrors errors = new GenericValidationErrors(pipeline);
-    pipelineValidators.forEach(it -> it.validate(pipeline, errors));
+    try {
```

Comment:
Leaving a comment so you can see this in a sea of files: here is where we throw a validation exception rather than adding to the errors object for the pipeline validators.